engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -435,6 +437,15 @@ public static String isValidImportString(Logger log, String importString) {
         }
     }
 
+    private static boolean visibleToClassGraph(String packageImport) {
+        try (ScanResult scanResult = new ClassGraph().enableAllInfo().acceptPackages(packageImport).scan()) {
```

Comment:
Is it possible that a user could run `import com.example.*.SomeClass`? This is something that `acceptPackages` allows, but `import <xyz>` doesn't (I don't think...).

Reply:
Doesn't look like it:
```
java.lang.RuntimeException: Attempting to import a path that does not exist: import groovy.*.TimeCategory;
```

Where as both of these do work:
```groovy
import groovy.time.*
import groovy.time.TimeCategory
```